CONTRIBUTING.md

```diff
@@ -15,10 +14,17 @@ Thank you for your interest in contributing to the Uniswap interface! 🦄
 1. `yarn install`
 1. `yarn build`
 
+## Running tests
+
+1. Run unit tests: `yarn test`
```

Comment:
These are all distinct, so they should be bullets, not numbered (`-`).

Reply:
Done